src/python/pants/backend/python/rules/targets_test.py

```diff
@@ -74,31 +68,10 @@ def test_timeouts_disabled(self) -> None:
 
 
 class TestPythonSources(TestBase):
-    PYTHON_SRC_FILES = ("f1.py", "f2.py")
-    PYTHON_TEST_FILES = ("conftest.py", "test_f1.py", "f1_test.py")
-
     @classmethod
     def rules(cls):
         return [*target_rules(), RootRule(HydrateSourcesRequest)]
 
-    def test_python_sources_validation(self) -> None:
```

Comment:
The deleted tests in this file are moved to more generalized tests in `engine/target_test.py`.